Makefile

```diff
@@ -125,23 +123,16 @@ oam-kubernetes-runtime.help:
 
 help-special: oam-kubernetes-runtime.help
 
-.PHONY: oam-kubernetes-runtime.help help-special docker-build docker-push kind-load e2e-setup e2e-test e2e-cleanup
-
-# Build the docker image
-docker-build:
-	docker build . -t $(IMG)
-
-# Push the docker image
-docker-push:
-	docker push ${IMG}
+.PHONY: oam-kubernetes-runtime.help help-special kind-load e2e-setup e2e-test e2e-cleanup
 
 # load docker image to the kind cluster
 kind-load:
-	kind load docker-image $(IMG) || { echo >&2 "kind not installed or error loading image: $(IMG)"; exit 1; }
+	docker tag $(IMAGE) crossplane/oam-kubernetes-runtime:$(VERSION)
```

Comment:
Can you point me to where/how is $(IMAGE) build?

Reply:
It is in `image.mk` in the `build/` submodule